gp-includes/things/translation-set.php

```diff
@@ -313,14 +316,15 @@ public function import( $translations, $desired_status = 'current' ) {
 
 				/**
 				 * Filter the the status of imported translations of a translation set.
+				 * Parameters $new_translation and $old_translation since 2.3.0.
```

Comment:
For changelog entries you can use the `@since` tag, see https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#since-section-changelogs.

```php
/**
 * … 
 * @since 1.0.0
 * @since 2.3.0 Added the `$new_translation` and `$old_translation` parameters.
 * …
 **/
```